features/rt/manifests/05-mc-kargs-worker-rt.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+    labels:
+        machineconfiguration.openshift.io/role: worker-rt
+    name: 10-kargs-worker-rt
+spec:
+    kernelArguments:
+        - default_hugepagesz=1G
+        - hugepagesz=1G
+        - hugepages=32
+#        - isolcpus=2-15
```

Comment:
Can we remove the commented line?

Reply:
That's why this PR is called "initial", I guess.  isolcpus or similar way of pining to CPU cores will be needed and this PR doesn't address this.

Reply:
in this case, we can remove it and address it in the following PR
In general, I think we can provide a template that will be filled with relevant values because of such parameters as a number of huge pages or CPU that we want to isolate can vary from node to node.